### ISSUE TITLE
Feature Request: Different color options for folders

### ISSUE THREAD

**Issue by NotoriousNico** [MoreWaita-Dev/MoreWaita#16]
First off: Thank you so much for having revived the MoreWaita icon project.

Here's my feature request:

Could you please add different color options for folders as variations to this theme?
Besides the default GNOME blue, I'm thinking of colors like Ubuntu's Yaru theme, Green or maybe even the accent colors from GNOME 47 as color variations for folders.

**Comment by Tywele** [comment]
I think having the GNOME 47 accent colours would be an excellent starting point.

**Comment by iaacornus** [comment]
> I think having the GNOME 47 accent colours would be an excellent starting point.

well, that's a good suggestion. Probably I'll put a presets using the defined colors of gnome's accent colors along with the popular themes e.g. nord, gruvbox and etc, and another option for custom folder colors. Although this will probably be a script that will modify the colors of existing folder icons, similar to papirus' folder color script

**Comment by NotoriousNico** [comment]
> well, that's a good suggestion. Probably I'll put a presets using the defined colors of gnome's accent colors along with the popular themes e.g. nord, gruvbox and etc, and another option for custom folder colors.

I've made the suggestion originally and really like your idea about having a script that will modify the colors of existing folders. That will not only give users more customization options, but it's also a way to have the icon pack integrate more natively with different OS, like Ubuntu or Linux Mint.

**Comment by iaacornus** [comment]
> > well, that's a good suggestion. Probably I'll put a presets using the defined colors of gnome's accent colors along with the popular themes e.g. nord, gruvbox and etc, and another option for custom folder colors.
> 
> I've made the suggestion originally and really like your idea about having a script that will modify the colors of existing folders. That will not only give users more customization options, but it's also a way to have the icon pack integrate more natively with different OS, like Ubuntu or Linux Mint.

yeah, it's now only matter to picking the color scheme. I'm also thinking of doing what papirus-icon-theme package does, a single package but installs different variant of the same theme to accomodate immutable distros where the scripts cannot modify system installation. I'll make a pr of this feature next week and see the best case
